In [1]:
# Das Paket tidyverse wird geladen
library(tidyverse)

── Attaching packages ─────────────────────────────────────── tidyverse 1.3.1 ──

✔ ggplot2 3.3.3     ✔ purrr   0.3.4
✔ tibble  3.1.1     ✔ dplyr   1.0.6
✔ tidyr   1.1.3     ✔ stringr 1.4.0
✔ readr   1.4.0     ✔ forcats 0.5.1

── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()



In [2]:
load(file = "Statistik2_Daten_Unterschiedshypothesen1.RData")


Wir Überprüfen ob der Datensatz "Statistik2_Daten_Unterschiedshypothesen1" geladen wurde.

Der Befehl head() zeigt dabei die ersten Spalten und Zeilen eines Datenframes an.

In [3]:
head(Statistik2_Daten_Unterschiedshypothesen1)

ID    gruppe sex vorwissen msk_1_prä msk_2_prä msk_2_prä_inv msk_3_prä
1 10101 0      1   10        4         1         4             4        
2 10102 1      1    4        4         1         4             4        
3 10103 0      1   11        4         1         4             4        
4 10107 1      1   10        4         1         4             4        
5 10108 0      0    9        3         4         1             4        
6 10109 1      1   15        4         1         4             4        
  msk_4_prä msk_5_prä ⋯ int_4_post int_5_post int_6_post int_post korr_prä
1 4         4         ⋯ 4          4          4          4.000000 0.0     
2 3         4         ⋯ 4          4          4          3.833333 0.0     
3 3         3         ⋯ 4          4          4          4.000000 0.5     
4 4         4         ⋯ 4          4          4          4.000000  NA     
5 3         3         ⋯ 4          4          4          4.000000 0.0     
6 4         4         ⋯ 4          4          4          4.000000 0.2     
  korr_post korr_fu   flex_prä flex_post flex_fu
1 0.3333333 0.0000000 0        4         1      
2 0.0000000 0.0000000 0        0         0      
3        NA        NA 1        0         0      
4 0.5000000 0.0000000 2        2         1      
5 0.4000000 0.0000000 0        1         0      
6 0.1666667 0.3333333 1        2         2

Die folgenden Übungsaufgaben beschränken sich auf die parametrischen Verfahren zur Überprüfung von
Unterschiedshypothesen – als auf t-Tests. 

Befehl: t.test()

Gebt jeweils die empirischen Kennwerte (Mittelwert, Standardabweichung, Teststatistik) sowie die
Interpretation der Ergebnisse an – so wird es von euch auch in der Klausur gefordert werden.

##1. Beispielaufgabe
Damit wir von einer Vergleichbarkeit der beiden Interventionsgruppen ausgehen können,
möchten wir überprüfen, ob sich die verschachtelte und die geblockte Bedingung (gruppe) in
ihrem Vorwissen (vorwissen) unterscheiden. 

Überprüft dies.

Solltest du nicht wissen wie ein Paket funktioniert kann man mit einem Befehl ? gefolgt von dem Paketnamen eine MAN-Page (Hilfeseite) anfordern.

In [ ]:
Zu aller erst wollen wir die Mittelwerte (M) und Standardabweichung (SD) der beiden Gruppen (gruppe) bezüglich des 
Vorwissens (vorwissen) ermitteln.

Hierfür benutzten wir aus dem Paket psych den Befehl describeBy

Wir haben schon gelernt das Befehle von Paketen mittels des Paketnamens (psych) gefolgt von einem :: und dann 
der gewünschte Befehl (describeBy) aufgerufen werden können.

In [13]:
psych::describeBy(x = Statistik2_Daten_Unterschiedshypothesen1$vorwissen, 
                  group = Statistik2_Daten_Unterschiedshypothesen1$gruppe)


 Descriptive statistics by group 
group: 0
   vars  n  mean   sd median trimmed  mad min max range  skew kurtosis   se
X1    1 98 13.82 5.78   14.5   14.04 6.67   0  25    25 -0.28    -0.63 0.58
------------------------------------------------------------ 
group: 1
   vars  n  mean   sd median trimmed  mad min max range skew kurtosis  se
X1    1 95 14.12 5.87     14   14.06 7.41   0  26    26 0.04    -0.79 0.6

Wir sehen das Gruppe 0 einen Mittelwert (mean) von M = 13.82 mit einer Standartabweichung (sd) von SD = 5.78 aufweist.
Gruppe 1 weist die Werte M = 14.12 und SD = 5.87 auf.

In [4]:
t.test(vorwissen ~ gruppe, data=Statistik2_Daten_Unterschiedshypothesen1)


	Welch Two Sample t-test

data:  vorwissen by gruppe
t = -0.35709, df = 190.57, p-value = 0.7214
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -1.953630  1.354704
sample estimates:
mean in group 0 mean in group 1 
       13.81633        14.11579 


Wir sehen das sich die beiden Gruppen nicht signifikant voneienader unterschieden, da der P-Wert (p-value) mit einem Wert von p = 0.7214 über unserem gewünschnten Signifikanznovau von p < 0.05 liegt. 

##2. Überprüft die Hypothese, dass die Schüler*innen nach der Intervention mehr unterschiedliche
Subtraktionsstrategien verwenden (flex_post) also vor der Intervention (flex_prä)